master/usage/calicoctl.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing and Configuring calicoctl
+---
+
+This document outlines how to install and configure calicoctl which is the
+primary tool for viewing, manipulating, and creating Calico objects on the
+command line.
+
+### Location, location, location
+
+The majority of calicoctl operations only need access to the datastore that
+Calico is using.  This means that the tool can be ran from any host 
+or container that can reach the datastore and has the proper credentials for
+access.
+
+#### Installing calicoctl
+
+The calicoctl tool can be downloaded from the
+[release page of the calicoctl repository]({{site.data.versions[page.version].first.components.calicoctl.url}}),
+set executable, and then it is ready to run.
+
+```
+curl -O -L {{site.data.versions[page.version].first.components.calicoctl.download_url}}
+chmod +x calicoctl
+```
+
+> *Note:* Move calicoctl to a directory in your PATH or add the directory
+  it is in to your PATH to avoid prepending the path to invocations of
+  calicoctl.
+
+#### "Installing" calico/ctl
+
+With each release of calicoctl the docker container `calico/ctl` is released to
```

Comment:
Running calicoctl as a container is outside the scope of this document. As far as user stories are concerned, the main reason people ever do it is when running it as a pod, which other docs cover (and this one can link to). 

If you're not running it as a pod, I don't think it buys you much to run it in a container. So best not to overcomplicate things here?